src/goto-programs/remove_returns.cpp

```diff
@@ -142,10 +142,14 @@ void remove_returnst::replace_returns(
 /// \param function_is_stub: function (irep_idt -> bool) that determines whether
 ///   a given function ID is a stub
 /// \param goto_program: program to transform
-void remove_returnst::do_function_calls(
+/// \return False if, and only if, instructions have been inserted. In that case
```

Comment:
Surely returning true if something has been done would be more intuitive? I appreciate there are various cbmc functions using the unix shell "zero = success" pattern, but I don't think this is a success/failure return.

Reply:
I couldn't decide... - so I'll happily take the guidance that you and @thk123 provided and will amend this.